js/scripts.js

```diff
@@ -7,137 +7,160 @@ let bodyElement = document.querySelector("#body");
 
 let teamsData = [];
 
-window.onload = function getTeamsData() {
+document.addEventListener("DOMContentLoaded", function getTeamsData() {
```

Comment:
I would suggest defining `function getTeamsData()` outside of the `DOMContentLoaded` event listener assignment just to keep the implementation similar across the app and of the off chance this function may be used somewhere else.

Reply:
Made this change